### PR TITLE
Switch to VS 2015

### PR DESCRIPTION
**Pull request by hrydgard**
This includes updates for the ffmpeg and native submodules, in separate pull requests.

### CODE REVIEW

File: Windows/main.cpp [hunk 1]
```diff
@@ -64,6 +64,15 @@ extern "C" {
...
 }
 
+#if 0
+#if defined(_MSC_VER) && _MSC_VER >= 1900
+extern "C" {
+// make dxerr.lib happy
+  int _vsnprintf(char *, const char *, size_t, ...);
+}
+#endif
+#endif
```
Comment (unknownbrackets) [inline Windows/main.cpp]: I think we can just delete this?

-[Unknown]

Comment (hrydgard) [inline Windows/main.cpp]: Uh, yeah, meant to get rid of that.

**Comment by hrydgard** [comment]
This will be merged once Orphis' build bot is ready, which should be in the next week.

**Comment by daniel229** [comment]
Looks good,have uninstalled vs2013 and installed vs2015,waiting it merged.

**Pull request merged**